ktor-server/ktor-server-core/jvm/src/io/ktor/server/http/ApplicationResponsePropertiesJvm.kt

```diff
@@ -2,9 +2,10 @@
  * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
  */
 
-package io.ktor.server.http
+package io.ktor.server.http.content
```

Comment:
package changed, but folder not, looks like not expected

Reply:
Thanks, will fix

Reply:
it should be in `io.ktor.server.response`